doc/langref.html.in

```diff
@@ -10851,28 +10851,26 @@ pub fn build(b: *std.Build) void {
     const run_step = b.step("run", "Run the app");
     run_step.dependOn(&run_cmd.step);
 
-    // Creates a step for unit testing. This only builds the test executable
-    // but does not run it.
-    const unit_tests = b.addTest(.{
+    const exe_unit_tests = b.addTest(.{
         .root_source_file = .{ .path = "src/main.zig" },
         .target = target,
         .optimize = optimize,
     });
 
-    const run_unit_tests = b.addRunArtifact(unit_tests);
+    const run_exe_unit_tests = b.addRunArtifact(exe_unit_tests);
 
     // Similar to creating the run step earlier, this exposes a `test` step to
     // the `zig build --help` menu, providing a way for the user to request
     // running the unit tests.
     const test_step = b.step("test", "Run unit tests");
-    test_step.dependOn(&run_unit_tests.step);
+    test_step.dependOn(&run_exe_unit_tests.step);
 }
       {#code_end#}
       {#header_close#}
 
       {#header_open|Building a Library#}
-      <p>This <code class="file">build.zig</code> file is automatically generated
-        by <kbd>zig init-lib</kbd>.</p>
+      <p>This <code class="file">build.zig</code> file is a subset of the file
+        automatically generated by the <kbd>zig init</kbd> command.</p>
```

Comment:
`init-exe` and `init-lib` were combined so now this repeats the same build.zig twice.

Reply:
The `build.zig` in the documentation are different, and only contains the bits either to generate a lib, or to generate an executable.

Reply:
Then could you update the documentation to match the output of `zig init`?

Reply:
But this is the output of `zig init` 😊

My intuition is that in order to create a library, after `zig init`, users will remove the bits relevant to creating an executable, and the other way round. This is exactly what was done here. 

Are you suggesting we merge the "creating a library" and "creating an executable" paragraphs into a single "creating a project" section?

Reply:
> Are you suggesting we merge the "creating a library" and "creating an executable" paragraphs into a single "creating a project" section?

Yes but if you think this is better maybe we should get a third opinion.